conf/symbiflow/environment.yml

```diff
@@ -3,6 +3,7 @@ channels:
   - defaults
   - litex-hub
 dependencies:
+  - litex-hub::yosys-uhdm=0.5_8401_g30b87e190
```

Comment:
Rethinking about this, I am worried that this yosys-uhdm replaces the yosys from the symbiflow package that is used for the VPR tests. 

I am not sure whether it would be better to actually have this in yet another separate environment

Reply:
``yosys-uhdm`` package uses ``PROGRAM_PREFIX`` and renames binary to ``uhdm-yosys``: https://github.com/hdl/conda-eda/blob/master/syn/yosys-uhdm/build.sh#L16

It should not replace yosys for other tests, but if you think it would be better to have separate environment for it, I can do it.

Reply:
All right, so, I believe that we can leave it as is, if the whole uhdm-yosys package lives separately from the original yosys one, I think we can leave it as is.